### PR TITLE
Spokenizer

### PR DESCRIPTION
**Pull request by whitney-mitchell**

### CODE REVIEW

**Comment by MeganWord** [comment]
hey! note on this: we're currently importing all our modules into Posterior.py so if we say python Posterior.py it'll run everything. right now you're importing domainizer/sentimentizer into your module, which works, but let's talk about the import structure tomorrow.

**Comment by MeganWord** [comment]
also! don't forget to write a docstring for anything that starts with 'def'!

File: Spokenizer.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+from Posterior import *
```
Comment (MeganWord) [inline Spokenizer.py]: ok, so I would remove this line... and then in posterior.py say from Spokenizer import *- and that would switch the import structure!

**Comment by MeganWord** [comment]
👍

**Comment by MelanieRBaker** [comment]
👍

**Comment by benwilburn** [comment]
👍

**Pull request merged**